Makefile

```diff
@@ -239,6 +239,27 @@ ct-mnesia-rocksdb: SYSCONFIG=config/test-lima.config
 ct-mnesia-rocksdb: AETERNITY_TESTCONFIG_DB_BACKEND=rocksdb
 ct-mnesia-rocksdb: internal-ct
 
+ct-stratum: KIND=test
+ct-stratum: SYSCONFIG=config/test.config
+ct-stratum: AETERNITY_TESTCONFIG_DB_BACKEND=mnesia
+ct-stratum: internal-build stratum-client-internal-build
+	@NODE_PROCESSES="$$(ps -fea | grep bin/aeternity | grep -v grep)"; \
+	if [ $$(printf "%b" "$${NODE_PROCESSES}" | wc -l) -gt 0 ] ; then \
+		(printf "%b\n%b\n" "Failed testing: another node is already running" "$${NODE_PROCESSES}" >&2; exit 1);\
+	else \
+		AETERNITY_TESTCONFIG_DB_BACKEND=$(AETERNITY_TESTCONFIG_DB_BACKEND) \
+		$(REBAR) ct --suite apps/aestratum/test/aestratum_integration_SUITE --sys_config $(SYSCONFIG); \
+	fi
+
+stratum-client-internal-build:
+	@(cd ./_build/$(KIND)/ && \
+	  if [ ! -d aestratum_client ]; then \
+	      git clone --depth 1 https://github.com/aeternity/aestratum_client; \
```

Comment:
In order to have reproduceable results, this checkout needs to be a specific commit. Where is the specific commit checked out?

Reply:
There isn't a specific version, but we can use HEAD of master as it is now: d9cff994107e8703

Reply:
I've added tag to aestratum_client, so now we pull a tagged version.